.github/workflows/release-new-version.yml

```diff
@@ -66,6 +66,17 @@ jobs:
         run: |
           git config --local user.name 'github-actions'
           git config --local user.email '41898282+github-actions[bot]@users.noreply.github.com'
+          # maybe we need to move this package.json as we need it in multiple workflows
+          npm install -g 'git+ssh://git@github.com:knutties/newman.git#37708c1fbf6b36240ac796a42dc5a7b435990764'
```

Comment:
@SanchithHegde  - I should possibly create a separate step for this - but I would like this commit to happen before the commit that `cog` generates.  Do you have a recommendation on how you want to place this code?

Reply:
We can split the git configuration, generating Postman collections, and performing the `cog bump` as three separate steps with suitable names, should be fine.

Reply:
Makes sense.  Done.